### PR TITLE
Anchors

### PR DESCRIPTION
**Pull request by Simon-Laux**

### CODE REVIEW

**Comment by Simon-Laux** [comment]
Functionally it works now.
It's not the prettiest, so if @Jikstra or anybody else wants to give the styling a try you're welcome to.
Here is the documentation/guide of the library I used: https://www.bryanbraun.com/anchorjs/

**Comment by hpk42** [comment]
@ralphtheninja @jikstra could you take a quick look if this PR looks roughly good? If not, then i am still fine with merging.  It's quite useful to be able to link into parts/subtitles of blog posts the website. thanks @Simon-Laux

File: assets/js/anchor.min.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...ns.icon),"always"===l&&(h.style.opacity="1"),""===this.options.icon&&(h.style.font="1em/1 anchorjs-icons","left"===this.options.placement&&(h.style.lineHeight="inherit")),"left"===this.options.placement?(h.style.position="absolute",h.style.marginLeft="-1em",h.style.paddingRight="0.5em",e[o].insertBefore(h,e[o].firstChild)):(h.style.paddingLeft="0.375em",e[o].appendChild(h))}for(o=0;o<u.length;o++)e.splice(u[o]-o,1);return this.elements=this.elements.concat(e),this},this.remove=function(A){for(var e,t,i=f(A),n=0;n<i.length;n++)(t=i[n].querySelector(".anchorjs-link"))&&(-1!==(e=this.elements.indexOf(i[n]))&&this.elements.splice(e,1),i[n].removeChild(t));return this},this.removeAll=function(){this.remove(this.elements)},this.urlify=function(A){return this.options.truncate||d(this.options),A.trim().replace(/\'/gi,"").replace(/[& +$,:;=?@"#{}|^~[`%!'<>\]\.\/\(\)\*\\\n\t\b\v]/g,"-").replace(/-{2,}/g,"-").substring(0,this.options.truncate).replace(/^-+|-+$/gm,"").toLowerCase()},this.hasAnchorJSLink=function(A){var e=A.firstChild&&-1<(" "+A.firstChild.className+" ").indexOf(" anchorjs-link "),t=A.lastChild&&-1<(" "+A.lastChild.className+" ").indexOf(" anchorjs-link ");return e||t||!1}}});
```
Comment (ralphtheninja) [inline assets/js/anchor.min.js:6]: No bitcoin miner included afaict :wink:

**Review by ralphtheninja — APPROVED** [review]
LGTM

**Comment by hpk42** [comment]
thanks @Simon-Laux !

**Comment by r10s** [comment]
cool, thanx @Simon-Laux

**Pull request merged**